console/cypress/.eslintrc

```diff
@@ -6,5 +6,13 @@
   "rules": {
     "no-unused-expressions": 0,
     "chai-friendly/no-unused-expressions": 2
-  }
+  },
+  "overrides": [
+    {
+      "files": ["./integration/validators/*"],
```

Comment:
`expect` and other expressions will throw an error if used outside `test.ts` or `spec.ts` files. We should either disable the rule in the file or through `.eslintrc` config.